cmd/mesh.go

```diff
@@ -11,7 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//go:generate go run ./vfsgen/vfsgen.go
+//go:generate
```

Comment:
Can the comment be removed since it's empty and not specifying anything? Or just a placeholder for something coming in a future PR?

Reply:
will remove thanks for catching.

Reply:
Actually, I am not sure if the comment can be removed.  I believe `// go generate` is needed to get vfsgen to operate on this file.

Reply:
Yes, this is a go generate directive and we have to keep it for the time being. Once the charts release process is working we will get rid of the built in charts.